### PR TITLE
byte-counter: add tip for CRLF

### PR DESCRIPTION
**Pull request by retornam**
@mathiasbynens this is for issue #25

### CODE REVIEW

File: byte-counter/index.html [hunk 1]
```diff
@@ -8,8 +8,9 @@
 <h1>UTF-8 string length & byte counter</h1>
 <noscript><strong>To use this tool, please <a href=http://enable-javascript.com/>enable JavaScript</a> and reload the page.</strong></noscript>
 <p><textarea autofocus>i ♥ u</textarea>
-<p>That’s <strong id=characters>5 characters</strong>, totaling <strong id=bytes>7 bytes</strong>. <a href=#%E2%82%AC id=permalink>#</a>
+<p>That’s <strong id="characters">5 characters</strong>, totaling <strong id="bytes">7 bytes</strong>. <a href="#%E2%82%AC" id="permalink">#</a>
```
Comment (michaelficarra) [inline byte-counter/index.html]: None of the other attributes are quoted; why include this in an unrelated PR?

Comment (retornam) [inline byte-counter/index.html]: @michaelficarra sorry. I'll fix this

File: byte-counter/index.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
 <p><textarea autofocus>i ♥ u</textarea>
 <p>That’s <strong id=characters>5 characters</strong>, totaling <strong id=bytes>7 bytes</strong>. <a href=#%E2%82%AC id=permalink>#</a>
 <p>Pro tip: add <code>https://mothereff.in/byte-counter#<var>%s</var></code> to the custom search engines / location bar shortcuts in your browser of choice. Whenever I want to check string length / byte count, I just enter <code>len <var>some string</var></code> in my address bar.
+<p>CRLF Issues: If you notice any carriage return or line feed issues please note that browsers normalize the input in form elements by default. See <a href=http://www.whatwg.org/specs/web-apps/current-work/multipage/the-button-element.html#the-textarea-element>spec</a> for detailed explanation.</p>
```
Comment (mathiasbynens) [inline byte-counter/index.html]: Updated link: https://html.spec.whatwg.org/multipage/forms.html#the-textarea-element

**Comment by retornam** [comment]
@mathiasbynens updated with new link

**Pull request closed without merging**